test/test_comparsion_generator.py

```diff
@@ -0,0 +1,101 @@
+import unittest
```

Comment:
There is a typo in the file name (it says comparsion)

Reply:
Yes, thanks!